### DISCUSSION TITLE
A song feature multiple singers

### DISCUSSION THREAD

**Discussion by AkimioJR** [navidrome/navidrome#2504] [Q&A]
**Fitstly,I am sorry for my poor English.Please relpy me in simple English.**
I search for HELP.
What can I do if a song or a album feature multiple singers and NAVIDROME can correctly each every singers.
What symbol should I use to separate each artist in the "artist" field of ID3 tags?

THANK FOR YOUR REPLY！
---------
**The following is my native language.**
我想在NAVIDROME中正确识别一首中的多个歌手，就是一首音乐进入多个歌手的页面
例如：有首歌的歌手分别是`A`和`B`，查看歌手详情后可以查看到`A`或`B`的所有歌曲，而不是查看到一个叫`A,B`的歌手的所有歌曲
我应该如何设置ID3标签中每位歌手间的符号
感谢各位的回答！

**Comment by metalheim** [comment]
As per id3 Standard, the NULL character shall be used ('\\\\' in Mp3tag) 

**But** Navidrome doesnt support multiple Artists per Song as of now.
I know @certuna is working on it, maybe He can advise whats best to use.

**Reply by AkimioJR** [reply]
> As per id3 Standard, the NULL character shall be used ('\\' in Mp3tag) 
> 
> 
> 
> **But** Navidrome doesnt support multiple Artists per Song as of now.
> 
> I know @certuna is working on it, maybe He can advise whats best to use. 

Thanks for your reply!

**Comment by certuna** [comment]
You can basically do three things:
- use the 'correct' id3v2.4 way, null-separated values: `artist` = "Artist1{null}Artist2"
- use a non-standard separator like the semicolon: `artist` = "Artist1;Artist2"
- use two tags: `artist` = "Artist 1 & Artist 2", `artists` = "Artist1{null}Artist2"

I've just submitted the PR to support multiple artists, all three of these methods.

**Reply by AkimioJR** [reply]
Looking forword to the new version
